sync/aws-parameter-store-manager-sync/src/main/java/co/com/bancolombia/secretsmanager/connector/AWSParameterStoreConnector.java

```diff
@@ -0,0 +1,92 @@
+package co.com.bancolombia.secretsmanager.connector;
+
+import co.com.bancolombia.secretsmanager.api.GenericManager;
+import co.com.bancolombia.secretsmanager.api.exceptions.SecretException;
+import software.amazon.awssdk.auth.credentials.*;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.ssm.SsmClient;
+import software.amazon.awssdk.services.ssm.SsmClientBuilder;
+import software.amazon.awssdk.services.ssm.model.GetParameterRequest;
+import software.amazon.awssdk.services.ssm.model.GetParameterResponse;
+
+import java.net.URI;
+import java.util.Optional;
+
+public class AWSParameterStoreConnector implements GenericManager {
+
+    private Region region;
+    private Optional<URI> endpoint;
+
+    public AWSParameterStoreConnector(String region) {
+        setRegion(region);
+        endpoint = Optional.empty();
+    }
+
+    /**
+     * This constructor allows make a connection for a local instance of
+     * AWS Parameter Store, such as: LocalStack, Docker container, etc.
+     *
+     * @param endpoint : String uri connection
+     * @param region   : Dummy region for Amazon SDK Client
+     */
+    public AWSParameterStoreConnector(String region, String endpoint) {
+        this.endpoint = Optional.of(URI.create(endpoint));
+        this.region = Region.of(region);
+    }
+
+    @Override
+    public String getSecret(String secretName) throws SecretException {
+        SsmClient ssmClient = buildClient();
+        return getSecret(secretName, ssmClient);
+
+    }
```

Comment:
can we implement this method?

Reply:
No, because Parameter Store only handle String type